### PR TITLE
Improve redis: health checks, no 'result' wrapper + proper exec

### PR DESCRIPTION
**Pull request by wilzbach**
Also, this redis service always returns string on `get`:

```
$ curl -X POST -d '{"key": "b", "value": 3}' -H "Content-Type: application/json" http://localhost:8000/set
$ curl -X POST -d '{"key": "b"}' -H "Content-Type: application/json" http://localhost:8000/get
"3"
```

Additionally, it fixes:

Fixes https://github.com/omg-services/redis/issues/22
Fixes https://github.com/omg-services/redis/issues/21
Fixes https://github.com/omg-services/redis/issues/20
Fixes https://github.com/omg-services/redis/issues/19
Fixes https://github.com/omg-services/redis/issues/17
Fixes https://github.com/omg-services/redis/issues/16
Fixes https://github.com/omg-services/redis/issues/15
Fixes https://github.com/omg-services/redis/issues/14
Fixes https://github.com/omg-services/redis/issues/13

### CODE REVIEW

**Comment by adnrs96** [comment]
I am not sure why @anukul closed his PR in favour on this. From what I understand this PR adds health  endpoint and anukul's PR was improving upon the output types.

**Comment by wilzbach** [comment]
This PR also changes the output type from a wrapped object to just the object itself. See the changes in `ok`.

**Comment by adnrs96** [comment]
I think we can in that case update the same in `microservice.yaml` as well.
I am talking about the `output` of  `get`. We should change it from `any`
to `string`?

**Comment by wilzbach** [comment]
> I think we can in that case update the same in `microservice.yaml` as well.
I am talking about the `output` of  `get`. 

Yeah, I was thinking about the same (hence the PR body) -> changed to `string`.

File: microservice.yml [hunk 1]
```diff
@@ -49,43 +57,140 @@ actions:
       type: any
   rpop:
     <<: *key_required
+    help: Removes and returns the last element of the list stored at 'key'.
     http:
       <<: *http
       path: /rpop
     output:
       type: any
   blpop:
```
Comment (wilzbach) [inline microservice.yml]: FWIW do we really need this command for an OMS service?

File: microservice.yml [hunk 2]
```diff
@@ -30,15 +35,18 @@ actions:
       type: none
   rpush:
```
Comment (wilzbach) [inline microservice.yml]: @judepereira can we name this and below friendlier for new users?

Comment (judepereira) [reply microservice.yml]: Are you referring to key_value_required?

File: microservice.yml (hunk 1 above)
Comment (judepereira) [reply microservice.yml]: Why not? There are use cases around it - holding a connection open for a while.

File: microservice.yml (hunk 2 above)
Comment (wilzbach) [reply microservice.yml]: No I was referring to `rpush` and friends. They aren't really in line with our recommended Storyscript naming that can be read as a proper sentence ;-)

Comment (judepereira) [reply microservice.yml]: Oh, no. I'd rather have the well known and standard redis commands to stay in line with the other redis clients out there.

**Pull request merged**